### PR TITLE
Testing/mhv-mr-refactor

### PR DESCRIPTION
**Pull request by mdewey**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No

## Summary

- Renamed the allergies e2e tests
- Updated the allergies e2e to align with the vitals e2e tests standards
- Cleaned up tests names
- moved all accelerated fixtures to the correct directory based on feedback from past PR 
- fixed typo in comment
- cleaned up and aligned vitals e2e tests
- added sessions data to fixtures

## Related issue(s)

Working in canvas in slack

## Testing done

- ensure the e2e ran

## Screenshots

N/A

## What areas of the site does it impact?

MHV - e2e tests

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

N/A

### CODE REVIEW

**Review by acrollet — DISMISSED** [review]

**Review by mattwrightva — DISMISSED** [review]
Nice!

**Review by IGallupSoCo — DISMISSED** [review]
LGTM

**Review by acrollet — APPROVED** [review]
re-approving after merge

**Review by mattwrightva — APPROVED** [review]

**Pull request merged**
